docs/admin/install.rst

```diff
@@ -0,0 +1,243 @@
+
+With the key and configuration available in ``dom0``, you're ready to set up SecureDrop Workstation:
+
+- First, re-enable the network connection using the network manager widget.
+
+- Next, start a terminal in the network-attached ``work`` VM, via **Q > Domain:work > work: Terminal**.
+
+- In the ``work`` terminal, run the following commands to download and add the SecureDrop signing key, which is needed to verify the SecureDrop Workstation package:
+
+  .. code-block:: sh
+
+    gpg --keyserver hkps://keys.openpgp.org --recv-key "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
+    gpg --armor --export 22245C81E3BAEB4138B36061310F561200F4AD77 | sudo tee /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation
+
+- In the ``work`` terminal, create a file ``/etc/yum.repos.d/securedrop-temp.repo`` with the following contents:
+
+  .. code-block:: none
+
+    [securedrop-workstation-temporary]
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation
+    enabled=1
+    baseurl=https://yum.securedrop.org/workstation/dom0/f25
+    name=SecureDrop Workstation Qubes initial install bootstrap
+
+- Download the SecureDrop Workstation config package with the command:
+
+  .. code-block:: sh
+
+    dnf download securedrop-workstation-dom0-config
+
+- In the ``dom0`` administration terminal, run the following command to transfer the RPM package to dom0 and install SecureDrop Workstation:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io work 'cat /home/user/securedrop-workstation-dom0-config-<versionNumber>-1.fc25.noarch.rpm' > securedrop-workstation.rpm
+    sudo dnf install securedrop-workstation.rpm
+
+  where ``<versionNumber>`` is the current SecureDrop Workstation release version. When prompted, press **Y** to install the package.
+ 
+Configure SecureDrop Workstation
+--------------------------------
+Before setting up the set of VMs used by SecureDrop Workstation, you must configure the Journalist Interface connection and submission key. 
+
+- To add the submission key, run the following command in the ``dom0`` terminal:
+
+  .. code-block:: sh
+
+    sudo cp /tmp/sd-journalist.sec /usr/share/securedrop-workstation-dom0-config/
+
+- Next, create the SecureDrop Workstation configuration file:
+
+  .. code-block:: sh
+   
+    cd /usr/share/securedrop-workstation-dom0-config
+    sudo cp config.json.example config.json
+
+- The ``config.json`` file must be updated with the correct values for your instance. Open it with root privileges in a text editor and update the following fields' values:
+
+  - **submission_key_fpr**: use the value of the submission key fingerprint
+  - **hidserv.hostname**: use the hostname of the Journalist Interface, including the ``.onion`` TLD
+  - **hidserv.key**: use the value of the v2 HidServAuth token for the Journalist Interface, or the v3 private authorization key value if your SecureDrop instance uses v3 onion services
+  - **environment**: use the value ``prod``
```

Comment:
Since that is the default, and there are other variables in the file, I would omit this from the list, and simply say that the other variables can be kept at their default values.

Reply:
Security implications if this, say, gets changed to dev or staging in the sample file in a later release. I would prefer to be explicit.